ui/main/src/app/business/view/activityarea/activityarea.view.spec.ts

```diff
@@ -238,7 +238,10 @@ describe('ActivityAreaView', () => {
         const activityAreaPage = await firstValueFrom(activityAreaView.getActivityAreaPage());
         expect(activityAreaPage.activityAreaClusters[0].lines).toHaveSize(2);
         expect(activityAreaPage.activityAreaClusters[0].lines[0].entityId).toEqual('ENTITY1');
-        expect(activityAreaPage.activityAreaClusters[0].lines[0].connectedUsers).toEqual(['anotherUser']);
+        expect(activityAreaPage.activityAreaClusters[0].lines[0].connectedUsers).toEqual([
+            'anotherUser',
+            'currentUser'
+        ]);
```

Comment:
the current test is to be removed as it was testing the fact that we remove the current user 

`it('GIVEN a user member of entity1 and entity2 WHEN user is currently connected to entity1 THEN entity1 line does not contains the current user login ',`

Reply:
Test not removed but the title of the test has been updated for the new behavior.